.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,13 +4,16 @@
 ## How can success be measured?
 
 
-## Screenshots (if applicable)
+## Screenshots
+<!--  If applicable, otherwise delete the header.
+      i.e. this is a visible frontend change -->
```

Comment:
I would maybe make it stronger, eg. _Required for any visible user-facing changes_? (requiring always before and after would be a tad too much, I guess)

Reply:
I don't think this needs additional explanation.

Reply:
Some PRs have been raised with images that didn't add much, hence the clarification. I don't think it needs to be stronger though. We can always push back if we think a particular PR warrants an image.